client/src/js/services/TransactionTypeStoreService.js

```diff
@@ -0,0 +1,43 @@
+angular.module('bhima.services')
+.service('TransactionTypeStoreService', TransactionTypeStoreService);
+
+// dependencies injection
+TransactionTypeStoreService.$inject = ['$q', 'TransactionTypeService', 'Store'];
+
+/**
+ * Transaction Type Store Controller
+ */
+function TransactionTypeStoreService($q, TransactionType, Store) {
+  'use strict';
+
+  var service = this;
+  var data = new Store();
+  var initialLoad = true;
+
+  var request = TransactionType.read()
+    .then(function (result) {
+      data.setData(result);
+      initialLoad = false;
+      return data;
+    });
+
+  service.data = transactionTypeStore;
```

Comment:
Instead of `data`, don't you think `load()` would be a better name?  Or even `get()`?